test/manual-scenarios/security/stop-network-polling-scenario.md

```diff
@@ -0,0 +1,24 @@
+# Stop Network Polling When MetaMask UI Is Closed
+
+## Feature Description
+
+This test scenario ensures that after closing the MetaMask UI, the extension stops sending network requests, including `etherscan`, `eth_call`, `eth_blockNumber`, and `eth_getBlockByNumber`, to Infura or any other RPC provider. This behavior is crucial for preventing unnecessary network traffic and ensuring privacy and efficiency.
```

Comment:
the account polling also throw this one: `eth_getBalance` , maybe we could also include it explicitly here?

see: https://github.com/MetaMask/metamask-extension/pull/26141/files#diff-174d8183603b0004a7ff4acb5a21374ea2ad88e63ab433090783ef41db9c35e7R153

Reply:
make sense, added in this commit [a754fb5](https://github.com/MetaMask/metamask-extension/pull/26195/commits/a754fb59c475e6c9b52d56079e99e208c2024bfd) thanks for the nice catch !